### DISCUSSION TITLE
[Feature] Allow to disable viewing of photos in an shared album

### DISCUSSION THREAD

**Discussion by TheZoker** [immich-app/immich#12498] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I know this may sound strange at first, but the idea here is, to just allow people to upload images without viewing them all the other already uploaded photos. 

A practical example of this would be a wedding. As a host I want to allow people to upload the photos, but I still want to review the photos before sharing them publicly. So next to the option of allowing people to download/upload photos for albums an additional option for hiding already uploaded photos would be nice 👍 

(this feature is similar to the file request feature of nextcloud) 

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#5681
